Sources/MessageIndex.php

```diff
@@ -339,11 +339,18 @@ function MessageIndex()
 		$message_index_tables = array();
 		$message_index_wheres = array();
 		call_integration_hook('integrate_message_index', array(&$message_index_selects, &$message_index_tables, &$message_index_parameters, &$message_index_wheres, &$topic_ids));
+		
+		if (!empty($modSettings['enableParticipation']) && !$user_info['is_guest'])
+			$enableParticipation = true;
+		else
+			$enableParticipation = false;
 
 		$result = $smcFunc['db_query']('substring', '
 			SELECT
 				t.id_topic, t.num_replies, t.locked, t.num_views, t.is_sticky, t.id_poll, t.id_previous_board,
 				' . ($user_info['is_guest'] ? '0' : 'COALESCE(lt.id_msg, COALESCE(lmr.id_msg, -1)) + 1') . ' AS new_from,
+				' . ( $enableParticipation ? ' COALESCE(( SELECT 1 FROM {db_prefix}messages parti WHERE t.id_topic = parti.id_topic and parti.id_member = {int:current_member} LIMIT 1) , 0) as is_posted_in,
```

Comment:
Please include the `AS` for consistency. 

`SELECT 1 FROM {db_prefix}messages parti` -> `SELECT 1 FROM {db_prefix}messages AS parti`

Reply:
the reason consistency is the only valid reason to do so.
because it not needed technicaly.